ui/app/src/components/shared/ResourceOperationListItem.tsx

```diff
@@ -14,15 +16,34 @@ export const ResourceOperationListItem: React.FunctionComponent<
       color: DefaultPalette.neutralSecondary,
     },
   };
+  const [isErrorPanelOpen, setIsErrorPanelOpen] = React.useState(false);
+
+  // Check if the value is an error message
+  const isError = props.val.includes("Error:") || props.val.includes("error:");
+
   return (
     <>
       <Stack wrap horizontal>
         <Stack.Item styles={stackItemStyles} style={{ width: "20%" }}>
           {props.header}
         </Stack.Item>
-        <Stack.Item styles={stackItemStyles} style={{ width: "80%" }}>
-          : {props.val}
-        </Stack.Item>
+
+        {isError ? (
+          <>
+            <Stack.Item styles={stackItemStyles} style={{ width: "80%" }}>
+              <Link onClick={() => setIsErrorPanelOpen(true)}>An error occurred, click to view the error details</Link>
```

Comment:
nit: Can you either make this two sentences, or use a semi-colon?
```suggestion
              <Link onClick={() => setIsErrorPanelOpen(true)}>An error occurred; click to view the error details</Link>
```